catalog-info.yaml

```diff
@@ -123,4 +123,37 @@ spec:
   lifecycle: production
   system: control-plane
 
+---
 
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: kibana-release
+  description: Definition of the kibana release pipeline
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/kibana-release
+spec:
+  type: buildkite-pipeline
+  owner: group:kibana-operations
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana-release
+      description: Pipeline that releases kibana by triggering the release flow through qa, staging, and production
+    spec:
+      repository: elastic/kibana
+      pipeline_file: ./.buildkite/pipelines/pipeline.kibana-release.yaml
+      provider_settings:
+        build_branches: false
+        build_pull_request_forks: false
+        build_tags: true
+        # https://regex101.com/r/tY52jo/1
+        filter_condition: 'build.tag =~ "/^deploy@\d+$$/"'
```

Comment:
Call out here that someone with kibana admin access might want to configure tag protection rules to prevent a release from being kicked off by an unauthorized person pushing a tag which matches the regex.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-tag-protection-rules

Reply:
It's a very good point/idea. I'd need to know, however who needs to be able to bypass these rules?

What's the plan? Are these tags going to be mostly created by automation (once the changes in the versions.yml is picked up)? Or is intended to be created manually when a release process is started?

Reply:
This will initiate a change to `versions.yaml`.  Part of the workflow triggered by the webhook is making a PR with the `BUILDKITE_COMMIT:0:12` on the gitops-repo which results in a delpoy of the commit associated with the created tag.

Right now I think its just a manual thing @watson was looking for.  Y'all can do whatever you want with this job.

Reply:
The intent is that these deploy tags should be created by a GitHub Action workflow. I've just created a PR that contains this workflow here: https://github.com/elastic/kibana/pull/165213

Reply:
All right, still, the rights for who can create `deploy@` tags, or further down, who can run this job need to be limited to avoid abuse.

Reply:
> I think yall might need to chase that up before merging?

No we can always put this protection in place later. It's not important right now as there's no customers live on our serverless infrastructure. But I'll make a note that we need to put this in place